config/environments/production.rb

```diff
@@ -118,12 +118,12 @@
     authentication: :plain,
     user_name: ENV["SENDGRID_USERNAME_ACCEL"],
     password: ENV["SENDGRID_PASSWORD_ACCEL"],
-    domain: "dev.to",
+    domain: ENV["APP_DOMAIN"],
     enable_starttls_auto: true
   }
 
   config.middleware.use Rack::HostRedirect,
-                        "practicaldev.herokuapp.com" => "dev.to"
+                        ENV["HEROKU_APP_URL"] => ENV["APP_DOMAIN"]
 end
 
-Rails.application.routes.default_url_options = { host: Rails.application.config.app_domain, protocol: :https }
+Rails.application.routes.default_url_options = { host: Rails.application.config.app_domain, protocol: ENV["APP_PROTOCOL"] }
```

Comment:
In the ENV file `APP_PROTOCOL` is actually "https://" and it looks like protocol is expected to just be "https", can you strip the extra characters to ensure the option is correctly formatted?

Reply:
Yeah. Protocol is actually also the wrong term here, so eventually we should probably rename this.

Reply:
Would you like me to submit a fix changing it to something like `APP_SCHEME` wherever it pops up?  Or a bunch of little fixes?

Maybe that sounds too dastardly...

![Patrick rubbing hands together evilly.](https://media3.giphy.com/media/Dps6uX4XPOKeA/giphy.gif)

`APP_URL_SCHEME`?  `APP_TRANSPORT_SCHEME`?  `APP_URL_PRELUDE`? `THAT_BIT_AT_THE_FRONT`?